src/main/kotlin/snyk/common/lsp/Types.kt

```diff
@@ -389,7 +389,7 @@ data class IssueData(
 
     // OSS
     @SerializedName("license") val license: String?,
-    @SerializedName("identifiers") val identifiers: OssIdentifiers?,
+    @SerializedName("identifiers") val identifiers: OssIdentifiers,
```

Comment:
How do we handle the case where OssIdentifiers are null and do we want the program to throw an exception if the value is null? 

"?": Safe call operator. Allows the program to continue without throwing an exception if the value is null.

Reply:
I don't remember why I changed this but I went back to see what happens if I add it back (since you are correct that it's safer to use in this case). It seems I can add it back. I feel more comfortable doing so because it has `omitempty` set in LS: https://github.com/snyk/snyk-ls/blob/0e5985de1531451cfcdd4788dfe98c8af79fd326/internal/lsp/message_types.go#L1052